docs/core/man/doveadm-stats.1.md

```diff
@@ -8,66 +8,105 @@ dovecotComponent: core
 
 ## SYNOPSIS
 
-**doveadm stats dump** [ **-s \<stats socket path\>** ] **\<type\>** [ **\<filter\>** ]
+**doveadm** [*GLOBAL OPTIONS*] **stats add** [ **-\-description** *\<string\>* ] [ **-\-exporter** *\<name\>* ] [ **-\-exporter-include** *\<field\>* ] [ **-\-fields** *\<fields\>* ] [**-\-group_by** *\<fields\>* ] *name* *\<filter\>*
 
-**doveadm stats top** [ **-s \<stats socket path\>** ] [ **-b** ] [ **sort** *\<field\>* ]
+**doveadm** [*GLOBAL OPTIONS*] **stats dump** [ **-s** *\<stats socket path\>* ] [ **-r** ] [ **-f** *\<fields\>* ]
 
-**doveadm stats reset** [ **-s \<stats socket path\>** ]
+**doveadm** [*GLOBAL OPTIONS*] **stats remove** [ *\<name\>* ]
 
-**doveadm stats add** [ **-\-description** *\<string\>* ] [ **-\-exporter** *\<name\>* ] [ **-\-exporter-include** *\<field\>* ] [ **-\-fields** *\<fields\>* ] [**-\-group_by** *\<fields\>* ] **\<name\>** *\<filter\>*
+**doveadm** [*GLOBAL OPTIONS*] **stats reopen**
 
-**doveadm stats remove** *\<name\>*
+## DESCRIPTION
 
-**doveadm stats reopen**
+Commands to inspect and edit Dovecot stats/metrics generation.
 
-## DESCRIPTION
+<!-- @include: include/global-options.inc -->
 
-**doveadm stats dump** is used to output statistics
+## COMMANDS
 
-**doveadm stats top** is used to monitor statistics
+### stats add
 
-**doveadm stats reset** is used to reset statistics
+**doveadm** [*GLOBAL OPTIONS*] **stats add** [ **-\-description** *\<string\>* ] [ **-\-exporter** *\<name\>* ] [ **-\-exporter-include** *\<field\>* ] [ **-\-fields** *\<fields\>* ] [**-\-group_by** *\<fields\>* ] *name* *\<filter\>*
 
-**doveadm stats add** is used to add metrics to statistics
+**doveadm stats add** is used to add metrics to statistics.
 
-**doveadm stats remove** is used to remove metrics from statistics
+#### OPTIONS
 
-**doveadm stats reopen** is used to reopen any file exporter files.
+**--description** *\<string\>*
+:   TODO
```

Comment:
`See [[setting,metric_description]]`

Reply:
Hmm, these don't exist in main (only in config-rewrite).

Irrelevant though, since we can't really use settings links in man pages anyway - since they don't mean anything in the man context.  So I'll just copy text over.